### PR TITLE
fixed a couple of bugs in the controller, and added to the create method

### PR DESCRIPTION
**Pull request by mcmillanator**

### CODE REVIEW

File: app/controllers/expenses_controller.rb [hunk 1]
```diff
@@ -1,14 +1,22 @@
-class ExpenseController < ApplicationController
+class ExpensesController < ApplicationController
```
Comment (mcmillanator) [inline app/controllers/expenses_controller.rb]: Controllers are suppose to be plural.  Singular was causing errors.

**Comment by jasimper** [comment]
Go team!

File: app/controllers/expenses_controller.rb [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
   def index
     respond_to do |format|
       format.html
-      format.json do 
+      format.json do
         @expenses = Expense.all
       end
+    end
```
Comment (mcmillanator) [inline app/controllers/expenses_controller.rb]: We were missing this end

File: app/controllers/expenses_controller.rb [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
   end
 
   def create
-    Expense.create(params(:expense))
+    @expenses = Expense.new(expense_params)
```
Comment (mcmillanator) [inline app/controllers/expenses_controller.rb]: Added the json for the create method.

**Pull request merged**